ui/app/ducks/swaps/swaps.js

```diff
@@ -376,9 +377,11 @@ export const fetchQuotesAndSetQuoteState = (
   metaMetricsEvent,
 ) => {
   return async (dispatch, getState) => {
+    const state = getState();
+    const chainId = getCurrentChainId(state);
     let swapsFeatureIsLive = false;
     try {
-      swapsFeatureIsLive = await fetchSwapsFeatureLiveness();
+      swapsFeatureIsLive = await fetchSwapsFeatureLiveness(chainId);
```

Comment:
I just noticed that this method is in a utility file that exists in pages/swaps/swap.util but is also used in the background. Could we move these shared utility methods into `shared/` ?

Reply:
I have started this on a separate branch that I will PR after the present PR is merged https://github.com/MetaMask/metamask-extension/pull/new/swaps-utils-shared-directory